ArjunaCore/arjuna/classes/com/arjuna/ats/arjuna/coordinator/BasicAction.java

```diff
@@ -2761,6 +2762,8 @@ protected int doPrepare (boolean reportHeuristics, AbstractRecord record)
                     * appear on multiple lists at the same time.
                     */
 
+                    addDeferredThrowables(record, deferredThrowables);
```

Comment:
The comment just above is now a little unclear. Move this call to before the comment.